LWOTC_WeaponPack/Localization/SMGs/XComGame.int

```diff
@@ -21,4 +21,11 @@ FriendlyName="Scramgun"
 FriendlyNamePlural="Scramguns"
 BriefSummary="The Scramgun, like the SMG and Stuttermag, is a smaller primary weapon that grants mobility and concealment bonuses, but does less damage and suffers accuracy penalties at longer ranges. Still, it is a personal weapon of significant power."
 TacticalText="<Bullet/> Firing the primary weapon always ends the soldier's turn\n<Bullet/> Use upgrades recovered from ADVENT enemies to improve the Scramgun's tactical effectiveness. \n<Bullet/> SMG-style weapons offer bonus mobility, close-quarters accuracy and concealment relative to assault rifles, but they do less damage, have less ammo, and suffer accuracy penalties at long range.<br/><Bullet/> Plasma weapons also cause targets to take 1 additional damage on future hits."
-AbilityDescName="SMG"
\ No newline at end of file
+AbilityDescName="SMG"
+
+[Carbine_CV X2WeaponTemplate]
+FriendlyName="Carbine"
+FriendlyNamePlural="Carbines"
+BriefSummary="Carbine!!! Submachine guns offer a lighter, more concealable alternative to the standard assault rifle, although they lack the same punch and are less accurate at long range."
+TacticalText="<Bullet/>Carbine!!! Firing the primary weapon ends the soldier's turn, in the absence of special training.\n<Bullet/> Use upgrades recovered from ADVENT enemies to improve the SMG's tactical effectiveness. \n<Bullet/> SMG-style weapons offer bonus mobility, close-quarters accuracy and concealment relative to assault rifles, but they do less damage, have less ammo, and suffer accuracy penalties at long range."
```

Comment:
Can you clean this up? You get full creative freedom to write what you want here.